### PR TITLE
chore(readme)

### PR DESCRIPTION
**Pull request by oded-soffrin**

### CODE REVIEW

**Comment by oded-soffrin** [comment]
@lxgreen @shaulgo made some cleanup,
(removed most of it, and pointed to ricos docs)

Looks good?

File: README.md [hunk 1]
```diff
@@ -7,211 +7,56 @@
...
 
-## Installation
+## Quick start
 
-To install this package as editor, use the following command
+Start from installing the package
 
 ```bash
-npm install --save wix-rich-content-editor 
+npm i ricos-editor
```
Comment (justame) [inline README.md:17]: no --save?

Comment (oded-soffrin) [reply README.md:17]: apparently save is on by default for a while now... you're a dinosaur like me :)

**Review by lxgreen — APPROVED** [review]
looks great! we also need a new gif

**Pull request merged**